news/_posts/2020-12-09-pr-testing-on-prestashop.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title:  "PR Testing on PrestaShop"
+subtitle: "How does the QA team handle the PR testing ?"
+date:   2020-12-08 16:00:00
+authors: [ RobinFischer ]
+icon: icon-airplane
+tags: [tests, community, contribute]
+---
+
+PrestaShop SA created a Quality Assurance team 4 years ago, in order to improve the quality of the open-source project, but also of the PrestaShop's products and services
```

Comment:
For information, the first paragraph will be displayed in the list, like below:
![capture d'écran_2895](https://user-images.githubusercontent.com/13449658/101648180-da99ee00-3a39-11eb-8175-0204f2169841.png)